pom.xml

```diff
@@ -89,7 +90,7 @@
                             <goal>shade</goal>
                         </goals>
                         <configuration>
-                            <createDependencyReducedPom>true</createDependencyReducedPom>
+                            <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
This will fix the issue the **pom.xml** file in source code is different than what is deployed(log4j-jboss-logmanager-XXX.pom) in maven repository. 

<code>mvn clean install </code> can be used to check the generated maven .pom file in local maven repository.(~/.m2/repository/org/jboss/logmanager/log4j-jboss-logmanager/)